.github/workflows/release.yml

```diff
@@ -17,8 +17,6 @@ jobs:
     steps:
       - uses: googleapis/release-please-action@v4
         id: release
-        with:
-          release-type: node
```

Comment:
this is a no-op change, but removing the `release-type` here will cause release-please to [use the config file instead](https://github.com/googleapis/release-please-action/blob/main/src/index.ts#L86), which will make it the source of truth.